micro_sam/training/joint_sam_trainer.py

```diff
@@ -13,6 +13,20 @@
 
 
 class JointSamTrainer(SamTrainer):
+    """Trainer class for jointly training the Segment Anything model with an additional convolutional decoder.
+
+    This class is inherited from `SamTrainer`.
+    Check out https://github.com/computational-cell-analytics/micro-sam/blob/master/micro_sam/training/sam_trainer.py
+    for details on its implementation.
+
+    Args:
+        unetr: The UNet-style model with vision transformer as the image encoder. Required to perform *AIS.
+        instance_loss: The loss to compare the predictions (for instance segmentation) and the targets.
+        instance_metric: The metric to compare the predictions and the targets.
+        kwargs: The keyword arguments of the `SamTrainer` (and `DefaultTrainer`) class.
+
+    NOTE: *AIS: Automatic Instance Segmentation
```

Comment:
I think we should not have this here. This will probably mess with the documentation generation. You can put this before `Args`.

Reply:
It's updated now. Thanks!